webui/react/package.json

```diff
@@ -35,7 +35,7 @@
     "query-string": "^6.13.5",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
-    "react-helmet": "^6.1.0",
```

Comment:
`react-helmet` is considered [inactive](https://github.com/nfl/react-helmet/issues/548#issuecomment-665351178)